### PR TITLE
Add Intel(R) Extension for Scikit-learn samples

### PR DESCRIPTION
**Pull request by PetrovKP**
# Existing Sample Changes
## Description

Add Intel(R) Extension for Scikit-learn samples into AI-and-Analytics/Features-and-Functionality folder for migration

Fixes Issue# 

## External Dependencies

List any external dependencies created as a result of this change.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] New feature (non-breaking change which adds functionality)

# Adding a New Sample(s)

## Description

Please include a description of the sample

## Checklist
Administrative
- [x] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): <insert Name Here>

Code Development
- [x] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [x] Adhere to readme template 
- [x] Enforce format via clang-format config file
- [x] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [x] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [x] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Security and Legal
- [x] OSPDT Approval - **pending** 
- [x] Bandit Scans (Python only)
- [x] Virus scan

Review
- [x] Review readme with Tom Lenth(@tomlenth) and/or Joe Oster(@JoeOster)
- [x] Tested using Dev Cloud when applicable

### CODE REVIEW

**Comment by PetrovKP** [comment]
@outoftardis , @racheloberman-intel can you check PR?

**Review by JoeOster — DISMISSED** [review]
approving for ci only

**Review by tomlenth — DISMISSED** [review]

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+## Running Samples on the Intel&reg; DevCloud
+If you are running this sample on the DevCloud, see [Running Samples on the Intel&reg; DevCloud](#run-samples-on-devcloud)
+
+## Build and Run the Sample
+
+### Pre-requirement
+
+> NOTE: No action is required if users use Intel DevCloud as their environment.
```
Comment (outoftardis) [inline AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md:34]:
```suggestion
> NOTE: No action is required if you are using Intel DevCloud as your environment.
```

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  Please refer to [Intel oneAPI DevCloud](https://intelsoftwaresites.secure.force.com/devcloud/oneapi) for Intel DevCloud.
```
Comment (outoftardis) [inline AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md:35]:
```suggestion
  Refer to [Intel oneAPI DevCloud](https://intelsoftwaresites.secure.force.com/devcloud/oneapi) for Intel DevCloud.
```

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+This sample code is implemented for CPU using the Python language. The example assumes you have Intel® Extension for Scikit-learn installed.
+
+## License  
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+## Running Samples on the Intel&reg; DevCloud
+If you are running this sample on the DevCloud, see [Running Samples on the Intel&reg; DevCloud](#run-samples-on-devcloud)
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/README.md:28]: @PetrovKP - please provide the correct url of your choice

**Review by JoeOster — CHANGES_REQUESTED** [review]

**Review by JoeOster — CHANGES_REQUESTED** [review]
As this is including a new dataset, I need information on this, name, URL and license

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+predicted = classifier.predict(x_test)
+report = metrics.classification_report(y_test, predicted)
+print(f"Classification report for SVC:\n{report}\n")
+
+# With scikit-learn-intelex patching you can:
+# 
+# - Use your scikit-learn code for training and prediction with minimal changes (a couple of lines of code);
+# - Fast execution training and prediction of scikit-learn models;
+# - Get the same quality;
+# - Get speedup more than **72** times.
```
[the remark this answers is not in the document]
Comment (JoeOster) [reply AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.py:69]: @outoftardis -Please note this comment is a little vague for me, please make changes as you require. Best method is to clone krills repo and make changes there. Then offer a PR to krill to accept. Please note everything I approved will need to be pulled into his repo, so you don't duplicate work. If questions about this process, please contact me via IM or @raoberman

**Review by JoeOster — DISMISSED** [review]
approved for ci only

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.ipynb [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "27b99b44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import fetch_openml\n",
+    "x, y = fetch_openml(name='a9a', return_X_y=True)"
```
[the remark this answers is not in the document]
Comment (PetrovKP) [reply AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.ipynb:33]: Add reference for dataset

File: AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.ipynb [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+   "source": [
+    "params = {\n",
+    "    'C': 100.0,\n",
+    "    'kernel': 'rbf',\n",
+    "    'gamma': 'scale'\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```
[the remark this answers is not in the document]
Comment (PetrovKP) [reply AI-and-Analytics/Features-and-Functionality/IntelScikitLearn_Extensions_SVC_Adult/IntelScikitLearn_Extensions_SVC_Adult.ipynb:98]: Added

**Review by JoeOster — APPROVED** [review]
approved for ci only

**Review by JoeOster — APPROVED** [review]

**Review by JoeOster — CHANGES_REQUESTED** [review]
Pending OSPDT Approval

**Review by JoeOster — APPROVED** [review]

**Pull request merged**